src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -322,6 +322,10 @@ input.inline {
     padding: 0;
 }
 
+.padding-7px {
```

Comment:
if i don't add any padding (`padding-0`)the warning box gets squished especially the info icon with the text. `padding-15px` on the other hand was a bit too large i thought. That's why i added a medium rule `padding-7px`. I thought we were using a similar pattern as space utilities atm `{property}{sides}-{size}`

Reply:
You can add a space before the text (and anyway in cases like this we should use `em` instead of `px`).

Anyway although no longer relevant, Bootstrap's spacing utilities are pre-defined to a few fixed sizes like `0`, `1`, `2`, while we explicitly state the amount. I suppose both have their pros and cons but the way we do right now gives the impression that we can just happily add any arbitrary amount we want to the CSS.